pgjdbc/src/test/java/org/postgresql/test/jdbc2/UpdateableResultTest.java

```diff
@@ -529,4 +530,27 @@ public void testMultiColumnUpdate() throws Exception {
     st.close();
   }
 
+  @Test
+  public void testMultipleSchemas() throws Exception {
+    PgConnection pgConnection = con.unwrap(PgConnection.class);
```

Comment:
This is unfortunate. Application-visible interface is `PGConnection`, not `PgConnection` class.
It could make sense to add `getSchema/setSchema` to `PGConnection`